src/lib/mixins.scss

```diff
@@ -4,8 +4,28 @@
 	flex: 1 1 0;
 }
 
+@mixin flex-auto-properties {
+	flex-basis: auto;
```

Comment:
According to the [flexLayout docs](https://github.com/angular/flex-layout/wiki/fxFlex-API) `auto` is an alias for `flex: <grow> <shrink> 100%`, so flex basis should be set to `100%` instead of auto I guess?

Reply:
Ok I'll update it.